cql3/statements/alter_keyspace_statement.cc

```diff
@@ -57,6 +70,17 @@ void cql3::statements::alter_keyspace_statement::validate(query_processor& qp, c
         }
         try {
             auto ks = qp.db().find_keyspace(_name);
+
+            if (ks.get_replication_strategy().uses_tablets()) {
+                const std::map<sstring, sstring>& current_rfs = ks.metadata()->strategy_options();
```

Comment:
That's incorrect place to validate RF values. Below the `auto new_ks = _attrs->as_ks_metadata_update(...)` is called that can replace RF numbers with one another. _That_ thing should be validated, not the user-provided raw options

Reply:
OK, moved down and using `new_ks` to verify RFs.